src/Development/Rattle/Server.hs

```diff
@@ -103,6 +104,8 @@ addCmdOptions new r@Rattle{options=o@RattleOptions{rattleCmdOptions=old}} =
 withRattle :: RattleOptions -> (Rattle -> IO a) -> IO a
 withRattle options@RattleOptions{..} act = withUI rattleUI (return "Running") $ \ui -> withShared rattleFiles rattleShare $ \shared -> do
     options@RattleOptions{..} <- rattleOptionsExplicit options
+    options@RattleOptions{..} <- maybe (return options) (\fp -> return options{rattleCmdOptions=rattleCmdOptions ++ [C.FileStdout fp, C.FileStderr fp]}) rattleOut
```

Comment:
Does this work? You end up trying to redirect all command outputs to a file, and also have the file open for writing. Maybe it works only on Windows or Linux, but not both?

Reply:
It works if we do the closing and reopening you mention below; not sure if there was another way to accomplish this